tests/qos/files/qos_params.th.yaml

```diff
@@ -1,5 +1,260 @@
 qos_params:
     th:
+        topo-t1-64-lag:
+            src_port_ids: [20,50]
```

Comment:
why are the multiple src ports needed? do you make use of both the src ports in a single test? if so, we should pick something on the same slice

Reply:
we only use one single src port int test, as below port list from real test:
```
dst_port_id=52;
dst_port_2_id=53;
dst_port_3_id=54;
src_port_id=20;
```
so using multiple src port from different xpe scenario won't occur.

The two ports defined here refer to other qos yml definitions of the same topology, providing a range for test code selection.